.vsts-dotnet-ci.yml

```diff
@@ -3,12 +3,28 @@ trigger:
 - exp/*
 - vs*
 
+variables:
+- group: AzureDevOps-Artifact-Feeds-Pats
+- name: cfsNugetWarnLevel
+  value: warn
+- name: nugetMultiFeedWarnLevel
+  value: none
+- name: NugetSecurityAnalysisWarningLevel
+  value: none
+
 jobs:
 - job: BootstrapMSBuildOnFullFrameworkWindows
   displayName: "Windows Full"
   pool:
     vmImage: 'windows-2022'
   steps:
+  - task: PowerShell@2
```

Comment:
We don't need these access to internal artifact feeds anymore. I think it is better if these are deleted from code.